markdown/appendices/resources.md

```diff
@@ -25,10 +25,25 @@ title: Other Doomsday Resources
 
 - [Legacy Breakfast](https://legacy-breakfast.com/tag/doomsday/)
 - [Eternal Central](https://www.eternalcentral.com/tag/doomsday/)
+- [The Dead Format episode 106 ft. Nevilshute](https://soundcloud.com/the-dead-format/episode-106-doomsday-with-nevilshute-martin-nielsen)
 
 ## Videos
 
+- [DK_TTGL runs a League with Grixis Meandeck (June 2020) - Portugese]
+  (https://www.youtube.com/watch?v=eMgfFlx0_10)
+- [Jarvis Yu ft. Ari Lax do a League with 4C Meandeck (July 2020)]
+  (https://www.youtube.com/watch?v=mn4GYFoVrRA)
 - [Anuraag Das's VOD featuring monkeyscantcry (march 2020)]
   (https://www.twitch.tv/videos/565641017)
 - [d8dk32's top 8 at the Legacy Titanium Series 1K Challenge (august 2017)]
   (https://www.youtube.com/watch?v=VmmR_3pG0Rs&t=38m3s)
+
+## Highlight Reel
+
+- [Monkeyscantcry shows the best pile to play around Surgical Extraction]
+  (https://www.twitch.tv/videos/564975441)
+- [Romariovidal finishes a league in spectacular fashion]
+  (https://www.twitch.tv/videos/698945172)
+- [Romariovidal builds the "perfect pile" on Esper Meandeck]
```

Comment:
@Doishy This one was fantastic, but the clip is too short :-(

Reply:
What do you mean too short?! YOU WANT MOAR?!

Reply:
I meant I can't imagine the uninitiated viewer will understand what was going on tbh.